pkg/userdata/flatcar/provider.go

```diff
@@ -430,3 +456,289 @@ storage:
 {{ safeDownloadBinariesScript .KubeletVersion | indent 10 }}
           systemctl disable download-script.service
 `
+
+// Coreos cloud-config template
+const userDataCloudInitTemplate = `#cloud-config
```

Comment:
Is it possible to simply merge values where both format are similar? for example, the units in coreos-cloudinit are one to one, to those in ignition. So why not simply have a var called `units` and `files` and add them to both templates.

Reply:
It's not possible as there are differences with naming both for units and files, like `content` vs `contents`, `mode` vs `permissions` and so on.